README.md

```diff
@@ -219,7 +219,7 @@ If the issue has been confirmed as a bug or is a feature request, please file a
 **Support Channels**
 
 * [New Relic Documentation](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs): Comprehensive guidance for using our platform
-* [New Relic Community](https://discuss.newrelic.com/tags/c/telemetry-data-platform/agents/nodeagent): The best place to engage in troubleshooting questions
+* [New Relic Community](https://forum.newrelic.com/): The best place to engage in troubleshooting questions
```

Comment:
this doesn't direct us to the Node.js agent tag.  We discussed in standup but wondering if we can generic short links